oaknational/oak-ai-lesson-assistant
# feat: add Prisma Accelerate caching to some of our queries

## Description

- Adds Prisma Accelerate caching to some of our queries.
- These are queries on tables which are write-once or only update during the chat.
- We could see how this performs in the Prisma Accelerate dashboard and then potentially consider some other tables which might benefit from caching.


## How to test

- Use the app to ensure that we aren't seeing any strange behaviour
- Check the Prisma Accelerate dashboard - we should see a higher percentage of cached queries if this is successful

![@Stef screenshotted this on 2024-08-29 at 14 24 56@2x](https://github.com/user-attachments/assets/fdc13d32-9884-4d9c-b649-3b548f1cd64a)
